tiagosatur/bluejack
# FEATURE | Testing with jest

**What this PR does?**
Test componets, Home, and New Pile pages

**How to test?**
Run the testing command in terminal and see the cases that were tested
All tests are in test.js files
